androidlibrary_lib/src/androidTest/java/org/opendatakit/fragment/AlertNProgessMsgFragmentMgerTest.java

```diff
@@ -78,11 +78,11 @@ public void testProgressDialogIsCreated() {
 
                 systemUnderTest.createProgressDialog(PROGRESS_DIALOG_TITLE, PROGRESS_DIALOG_MESSAGE, fragmentManager);
                 fragmentManager.executePendingTransactions();
-                Fragment frag = fragmentManager.findFragmentByTag(PROGRESS_DIALOG_TAG);
```

Comment:
same here. testing in small steps helps pinpoint issues quickly.

Reply:
> same here. testing in small steps helps pinpoint issues quickly.

done too.